src/statusnotifieritem/statusnotifieritem.cpp

```diff
@@ -287,7 +287,8 @@ IconPixmapList StatusNotifierItem::iconToPixmapList(const QIcon& icon)
     IconPixmapList pixmapList;
 
     // long live KDE!
-    for (const QSize &size : icon.availableSizes())
+    const QList<QSize> sizes = icon.availableSizes();
+    for (const QSize &size : const_cast<QList<QSize>&>(sizes))
```

Comment:
Hm... is this correct?
You are casting the `const`-ness away... so the detach does happen here.

IMO the correct handling should be:
- either
```c++
const QList<QSize> sizes = icon.availableSizes();
for (const QSize &size : sizes)
```
- or
```c++
for (const QSize &size : const_cast<QList<QSize> const &&>(icon.availableSizes()))
```